### PR TITLE
Feature Home : Edit/Delete Post screens

### LINKED ISSUE #123: Create XML files for Edit/Delete Post screens
We need to create XML files for Edit/Delete Post designs. Here are the designs: https://zpl.io/beQQAdB

### PR DESCRIPTION
**Pull request by anurjalal**
What this PR does:  **Create XML files for Edit/Delete Post screens**

Which issue(s) this PR fixes:

**Fixes #123**

Special notes for reviewers: none

Additional comments: none

the designs: https://zpl.io/beQQAdB

**I'm testing button in runtime.**
**edit_delete_fragment.xml**  -   when delete button is pressed :

cek1

**edit_delete_fragment.xml**  -  when no button is pressed :

cek2

**delete_post_alert.xml**  :

dialogue

**delete_post_feedback.xml**  :

post_deleted

### CODE REVIEW

**Review by mayorjay — APPROVED** [review]

File: .idea/markdown-navigator-enh.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+    <HtmlExportSettings updateOnSave="false" parentDir="" targetDir="" cssDir="css" scriptDir="js" plainHtml="false" imageDir="" copyLinkedImages="false" imagePathType="0" targetPathType="2" targetExt="" useTargetExt="false" noCssNoScripts="false" useElementStyleAttribute="false" linkToExportedHtml="true" exportOnSettingsChange="true" regenerateOnProjectOpen="false" linkFormatType="HTTP_ABSOLUTE" />
+    <LinkMapSettings>
+      <textMaps />
+    </LinkMapSettings>
+  </component>
+</project>
```
Comment (carl05) [inline .idea/markdown-navigator-enh.xml]: pls remove these files and check if .idea is on gitignore

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -19,6 +19,9 @@
       </option>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
+    <MarkdownNavigatorCodeStyleSettings>
+      <option name="RIGHT_MARGIN" value="72" />
+    </MarkdownNavigatorCodeStyleSettings>
     <codeStyleSettings language="XML">
       <indentOptions>
         <option name="CONTINUATION_INDENT_SIZE" value="4" />
```
Comment (carl05) [inline .idea/codeStyles/Project.xml]: pls remove these files and check if .idea is on gitignore

File: .idea/markdown-navigator.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    </HtmlSettings>
+    <CssSettings previewScheme="UI_SCHEME" cssUri="" isCssUriEnabled="false" isCssUriSerial="true" isCssTextEnabled="false" isDynamicPageWidth="true">
+      <StylesheetProvider>
+        <provider providerId="com.vladsch.md.nav.editor.text.html.css" providerName="No Stylesheet" />
+      </StylesheetProvider>
+      <ScriptProviders />
+      <cssText />
+      <cssUriHistory />
+    </CssSettings>
+  </component>
```
Comment (carl05) [inline .idea/markdown-navigator.xml]: pls remove these files and check if .idea is on gitignore

File: home/src/main/res/layout/delete_post_alert.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#6b000000">
```
Comment (carl05) [inline home/src/main/res/layout/delete_post_alert.xml:6]: use color.xml attribute values

File: home/src/main/res/layout/delete_post_alert.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="280dp"
+        android:layout_height="192dp"
```
Comment (carl05) [inline home/src/main/res/layout/delete_post_alert.xml:10]: use dimens.xml

File: home/src/main/res/layout/edit_delete_fragment.xml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        style="@style/buttonDelete"
+        android:layout_width="320dp"
+        android:layout_height="45dp"
+        android:layout_marginTop="8dp"
+        android:text="@string/delete_post"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/btn_edit_post" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (carl05) [inline home/src/main/res/layout/edit_delete_fragment.xml:41]: use dimens and colors

File: home/src/main/res/layout/delete_post_feedback.xml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        android:text="@string/deleted_msg"
+        android:textColor="@color/colorWhite"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
```
Comment (carl05) [inline home/src/main/res/layout/delete_post_feedback.xml:22]: use dimens and colors

File: home/src/main/res/layout/delete_post_alert.xml [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+            android:paddingBottom="9dp"
+            android:text="@string/delete"
+            android:textColor="@color/colorRed"
+            app:layout_constraintBottom_toBottomOf="@+id/btn_cancel"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/btn_cancel"
+            app:layout_constraintTop_toTopOf="@+id/btn_cancel" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+</androidx.coordinatorlayout.widget.CoordinatorLayout>
```
Comment (carl05) [inline home/src/main/res/layout/delete_post_alert.xml:74]: use dimens and colors

**Review by carl05 — CHANGES_REQUESTED** [review]
please solve comments

**Comment by Osaigbovo** [comment]
Hello Ahmad! Can you please delete this pull request and create it for the home-feed branch instead?

**Pull request closed without merging**